stories/filters.stories.js

```diff
@@ -323,9 +409,8 @@ ListFilter.parameters = {
   #Sample Code
 
   \`\`\`html
-  ${list_filter_html}
+  ${ListFilterHtml}
```

Comment:
Let's change this back to `${list_filter_html}` to keep things consistent with other stories